### ISSUE TITLE
[Bullet] Tooltips shouldn't appear on the entire canvas

### ISSUE THREAD

**Issue by ghudgins** [elastic/elastic-charts#1187]
**Is your feature request related to a problem? Please describe.**
Distracting to have the tooltip appear when you hover over white space in the chart canvas

**Describe the solution you'd like**
Consider only letting the "Actual" value show when hovering on the bullet 

**Kibana Cross Issues**
related to this integration: https://github.com/elastic/kibana/issues/89859

**Checklist**

Delete any items that are not applicable to this feature request.

- [x] this request is checked against already exist requests
- [x] every related Kibana issue is listed under `Kibana Cross Issues` list
- [x] `kibana cross issue` tag is associated to the issue if any kibana cross issue is present

**Comment by monfera** [comment]
Documenting some design considerations here:

- It's possible, and sometimes desirable to style a horizontal or vertical bullet such that it's thin (eg. for a clean look) eg. [this](https://elastic.github.io/elastic-charts/?path=/story/goal-alpha--horizontal-bar) or thinner
- Fitts law prompts us to have a decent sized mouse capture zone for the tooltip to activate; while in Lens config, the bullet chart may be in a huge area with hundreds of pixels of empty space around it, in a dashboard the entire thing may be a few tens of pixels horizontally or vertically; so it's useful to take the maximum of the computed width/height and some value at which the tooltip-on-background is not disturbing (something like 100px CSS)
- In case of circly gauge/goal (should they become requested by customers) it'd be esp. annoying to have to precisely hit a potentially thin circle (or tick label)
- due to tick placement, other annotations, ticks etc. the chart has no definite area

So I'm planning with something like this:

The width of the capture zone would be the largest of
- 100 CSS pixels (Fitts law driven)
- 50% of the chart width (so a decent fraction of the full chart area is tooltip sensitive)
- the width of the bounding box of anything that gets rendered

Same for height.

@markov00 @ghudgins  please bring up alternatives

**Comment by markov00** [comment]
@monfera I think this is a sensible selection of params, so I'm ok with your suggestion.
One alternative could be just the bounding box of the bullet chart without the label

**Comment by markov00** [comment]
or the background area of the quantitative scale independently from the size and weight of the bar

**Comment by ghudgins** [comment]
I agree with the concern (thinness) - great catch. I don't have any new suggestions: the bounding box idea sounds simplest but having a little bit of padding seems nice for usability. This is probably one of those things that needs to be quickly prototyped to zero in on the optimal **balance between easy to activate and easy to NOT activate**.
